quickwit/quickwit-doc-mapper/src/default_doc_mapper/default_mapper.rs

```diff
@@ -133,8 +133,8 @@ fn validate_timestamp_field(
     Ok(())
 }
 
-impl From<DefaultDocMapper> for DefaultDocMapperBuilder {
-    fn from(default_doc_mapper: DefaultDocMapper) -> Self {
+impl From<DocMapper> for DefaultDocMapperBuilder {
```

Comment:
And we need to rename `DefaultDocMapperBuilder` too?

Reply:
renamed to `DocMapperBuilder`